.github/ISSUE_TEMPLATE/pull_request_template.md

```diff
@@ -1,10 +1,3 @@
-* **Please check if the PR fulfills these requirements**
```

Comment:
according to https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository the file can be removed

Reply:
noted